templates/simple.html

```diff
@@ -1,9 +1,48 @@
-<section data-component="n-newsletter-signup" data-newsletter-id="{{id}}" data-newsletter-name="{{name}}" class="n-newsletter-signup n-newsletter-signup--simple" {{#if isPremium}}data-newsletter-is-premium{{/if}}>
-	<span class="n-newsletter-signup__description">Sign up to the {{name}} newsletter, {{frequency}} </span>
-
-	{{#if userNeedsToUpgrade}}
-		<a href="/products?location=/newsletters" class="n-newsletter-signup__upgrade" data-trackable="newsletters-upgrade">Upgrade<span class="o-normalise-visually-hidden">&nbsp; your subscription to receive {{name}}</span></a>
-	{{else}}
-		{{> n-newsletter-signup/templates/partials/newsletter-form}}
-	{{/if}}
+<section data-component="n-newsletter-signup"
+    data-newsletter-id="{{id}}" 
+    data-newsletter-name="{{name}}" 
+    class="n-newsletter-signup n-newsletter-signup--simple" 
+    {{#if isPremium}}data-newsletter-is-premium{{/if}}
+>
+    {{#if imageUrl}}
+    <div class="newsletter__flex-child-left">
+        <picture>
+            <source srcSet="{{imageUrl}}" />
+            <img
+                src="{{imageUrl}}"
+                alt=""
+                data-component="newsletter-picture"
+            />
+        </picture>
+    </div>
+    {{/if}}
+    <div class="n-newsletter-signup__card_content">
+        <div>
+            <h2 class="n-newsletter-signup__heading">
+                {{name}}
+                <span class="o-normalise-visually-hidden">
+                    {{#if isPremium}}, premium{{/if}}
+                    {{#if frequency}}, {{frequency}}{{/if}}
+                </span>
+            </h2>
+            
+            {{#if isPremium}}
+                <span aria-hidden="true" class="n-newsletter-signup__top-meta o-labels o-labels--content-premium">Premium</span>
```

Comment:
**Question:** Why is this marked as hidden via `aria-hidden="true"`? It appears to have styles applied to it that doesn't suggest its hidden

Reply:
My understanding of `aria-hidden="true"` is that it's used primarily for aesthetic purposes, to hide elements from assistive technologies that don't need to be read aloud or interacted with, which is why it's set to true. I didn't set this personally because it was there on the existing design which i still maintained it.

Reply:
Your understanding is correct but if its visible to sighted users then its likely it should be present for users who are using assistive tech. Why would a user who can't visually see the page not need to know this is a Premium newsletter? Do they get that information from somewhere else?

Reply:
This i am not aware but i don't think they are getting it somewhere. I will remove it.

Reply:
Looks like the reason its marked as `aria-hidden` is that the data is also in the headline so doesn't need to be duplicated.

Personally this looks a bit of a messy scenario but you can probably continue without changing this for now

![Screenshot 2024-02-12 at 14 05 48](https://github.com/Financial-Times/n-newsletter-signup/assets/524573/5df7a3bb-0b8e-4baf-83f6-41bded825880)

Reply:
That makes sense to me now, too. However, since the current design does not capture any such data, it would be reasonable to remove it to ensure screen readers can identify it as premium content.

Reply:
> However, since the current design does not capture any such data, it would be reasonable to remove it to ensure screen readers can identify it as premium content.

Can you explain this to me please I am not sure I understand what you mean here?

Reply:
In the previous design, the word `premium` appeared as text within the newsletter's content, but it is no longer present in the current design. This is why I mentioned earlier that it makes sense to remove the aria label now. Initially, the removal was to prevent screen readers from announcing it twice, but this concern no longer applies.

Reply:
Sorry I don't think my previous response with the screenshot was clear. In your new design the word premium is still included twice. Its in the title its just visually hidden. So it does still make sense to `aria-hidden` one of those 2 instances

Reply:
I see what you mean.